z/flags.go

```diff
@@ -247,3 +249,34 @@ func (sf *SuperFlag) GetString(opt string) string {
 	}
 	return sf.m[opt]
 }
+
+func (sf *SuperFlag) GetPath(opt string) string {
+	p := sf.GetString(opt)
+	path, err := expandPath(p)
+	if err != nil {
+		log.Fatalf("Failed to get path: %+v", err)
+	}
+	return path
+}
+
+// expandPath expands the paths containing ~ to /home/user. It also computes the ablosule path
+// from the relative paths. For example: ~/abc/../cef will be transformed to /home/user/cef.
+func expandPath(path string) (string, error) {
+	if len(path) == 0 {
+		return "", nil
+	}
+	if path[0] == '~' {
```

Comment:
This does not take into account filenames that start with `~`, eg. `"~myfilename"`.

On UNIX,
```go
path[0] == '~' && (len(path) == 1 || path[1] == '/')
```

On Windows,
```go
path[0] == '~' && (len(path) == 1 || path[1] == '/' || path[1] == '\\')
```

Reply:
We can actually create file with names `'~\\filename'` in unix. Which will cause problems if we include windows fix. I have fixed this for unix, but not for windows.

Reply:
@ahsanbarkati you can use `os.IsPathSeparator` to do this in a cross-platform manner.

Reply:
Nice. Thanks. Done :)